setup_python_locally.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
please put this part into a separate PR

Reply:
Done: https://github.com/Public-Tree-Map/public-tree-map-data-pipeline/pull/83